src/ontology/mondo-edit.obo

```diff
@@ -135422,7 +135422,7 @@ xref: MedDRA:10062804 {source="Orphanet:377/e", source="Orphanet:377"}
 xref: MESH:D001478 {source="DOID:2512", source="MONDO:equivalentTo"}
 xref: NCIT:C2892 {source="DOID:2512", source="MONDO:equivalentTo"}
 xref: NORD:1507 {source="MONDO:NORD"}
-xref: OMIM:109400 {source="Orphanet:377/e", source="DOID:2512", source="MONDO:equivalentTo", source="Orphanet:377"}
+xref: OMIMPS:109400 {source="MONDO:equivalentTo"}
 xref: Orphanet:377 {source="OMIM:109400", source="MONDO:equivalentTo"}
```

Comment:
if you turn this into the OMIMPS rather than the OMIM term, what happened to the OMIM term? did you check that all of the xrefs are equivalent to the OMIMPS and not OMIM:109400? This is normally a case where we would split this term (or [bud it](https://mondo.readthedocs.io/en/latest/editors-guide/splitting-classes/#splitting-a-more-generic-term-into-a-more-specific-terms_1))

Reply:
In all these cases, here OMIM:109400, a new term will be created during the next omim slurp/alignment, which I am planning on making as soon as this PR is merged, and before we release. So these OMIM terms will not be missing from Mondo from a user perspective. 

The reasoning behind it is that all of these new OMIMPS were created because there is a type 1 (previous OMIM) and type 2. So I want to make sure that we have both types instead of having a version where we have only 1 type (and having to add "single child" exclude from QC )

Reply:
Another way to see this is that this term would be split, but all the other x-refs and term ID would be maintained because it is referring to the higher concept. The "new term" in the split has not been created yet. 
(note: I am doing it this way only because I know that I will run the pipeline and get these "new terms" before the release. I wouldn't do it otherwise).

Reply:
All of these terms are reported in [this spreadsheet-Sheet1](https://docs.google.com/spreadsheets/d/1A7ILok6Ae7mHQPD88E8k_dGxqLcKcqKWmN0hXpeItXU/edit#gid=0), in yellow, with the "check after next round" follow-up.

Reply:
If I am understanding correctly, you will be changing the ID of the term that is equivalent to OMIM:109400, and MONDO:0007187 will no longer mean the disease that has a variation in a specific gene but to a group of diseases. It looks like this term conflates the grouping class and subclass (there are two DO xrefs). I think the correct approach here is to split this class, I personally don't agree with your proposed approach.

Reply:
I'd recommend consulting with Chris on this.

Reply:
Currently, the concept for MONDO:0007187 (and other similar examples) already represents a group of diseases and does NOT represent a specific disease associated with a variation in a gene. 
In all these examples, if we were to do in manually, we would keep the ID for the general term and create a new term for the specific disease represented by the OMIM id. 
I am doing the same thing here, except that the creation of the new, more specific term will be added in the next PR. 

@cmungall can you check this discussion and share your thoughts? 
If we need to discuss it on a call, that's also ok, but please let us know. 

(note: depending on the timing of the discussion and the decision, the OMIM ingest might need to wait until the next release)

Reply:
> Currently, the concept for MONDO:0007187 (and other similar examples) already represents a group of diseases and does NOT represent a specific disease associated with a variation in a gene.

some of the equiv xrefs refer to the disease with the gene variation though:
- https://www.ebi.ac.uk/ols4/ontologies/doid/classes?obo_id=DOID%3A0070365
- https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C2892

Reply:
> In all these examples, if we were to do in manually, we would keep the ID for the general term and create a new term for the specific disease represented by the OMIM id.

in the past, we have always obsoleted this term and created two new terms with two new IDs (split)

Reply:
the split process is definitely tedious. If we move away from it, I think we should definitely inform users about what is happening though because if someone is using the ID MONDO:0007187 with the understanding that it is equivalent to NCIT:C2892, which is the more specific term, then their annotation would become wrong.

Reply:
I think we need to discuss on a call, because I disagree with the statement "ID MONDO:0007187 with the understanding that it is equivalent to NCIT:C2892" (and I also disagree that NCIT:C2892 id actually more specific, but that's a different discussion). 

IMO, this is a case of ["Splitting a more generic term into a more specific term(s)"](https://mondo.readthedocs.io/en/latest/editors-guide/splitting-classes/#splitting-a-more-generic-term-into-a-more-specific-terms_1), and therefore we have already been doing this. 
This is based on the fact that the ontology term = the concept=the definition, and not the mappings. 
If we follow this logic, than we would have to obsolete Mondo terms every time we have incorrect mappings.
We do obsolete terms when the concept itself (definitions and axioms) was confusing. But I don't think the concept in these cases was confusing. 

I am not worried about the split process being tedious. But this means that we will not have an OMIM update for another 2 months.

Reply:
Sounds good. Good point about ontology term = the concept=the definition, this makes sense to me. There might be another case below where a logical axiom was deleted though, I'll review that and comment below.